packages/cli/src/lib/decorators/preconditions/authenticated.ts

```diff
@@ -1,11 +1,23 @@
 import Command from '@oclif/command';
+import {Config} from '../../config/config';
 import {
   AuthenticationStatus,
   getAuthenticationStatus,
 } from '../../platform/authenticatedClient';
 
 export function IsAuthenticated() {
   return async function (target: Command) {
+    const cfg = new Config(global.config.configDir);
+
+    const {organization} = await cfg.get();
+
+    if (!organization || organization.length === 0) {
```

Comment:
Use optional chaining to simplify your code:
```suggestion
    if (organization?.length) {
```